tools/test/examples/README.md

```diff
@@ -0,0 +1,81 @@
+# Examples testing script
+
+examples.py is script which makes easier to test examples.
+```
+usage: examples.py [-h] [-c CONFIG] [-e EXAMPLE] [-p TEST_PHASE]
```

Comment:
Looks like this usage stuff is repeated in the actual code. Is it needed here too?

Reply:
Yes it is copy paste from commandline. Does not make any harm to have it here

Reply:
Which is the authoritative source? By putting this in the documentation too it makes extra maintenance work.

Reply:
While I see @mark-edgeworth point , it can be useful in the readme providing more detailed information concerning the parameters is provided compared to what is shown on the help in the code...otherwise, yes it is just a duplicate

Reply:
> extra maintenance work.

+1

--help should provide all information, we dont need to copy-paste it here.